src/stories/Library/article-header/article-header.scss

```diff
@@ -0,0 +1,38 @@
+.article-header {
+  display: grid;
+  gap: $s-xl;
+  padding: $s-4xl $s-xl;
+  position: relative;
+
+  @include breakpoint-s {
+    padding: $s-4xl $s-7xl;
+  }
+
+  &__back-link {
+    position: absolute;
+    left: $s-md;
+    top: $s-sm;
+
+    @include breakpoint-s {
+      left: $s-2xl;
+      top: $s-lg;
+    }
+  }
+
+  &__title,
+  &__subtitle {
+    @include breakpoint-s {
+      max-width: 897px;
+    }
+  }
+
+  &__info {
+    display: flex;
+    align-items: baseline;
+    gap: $s-xs;
+
+    &__date {
```

Comment:
This will result in .article-header__info__date, which is too specific.
Either, make it article-header__date, or .article-header__info-date